src/main/java/frc/robot/subsystems/NFRBeamBreak.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems;
+
+
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+
+public class NFRBeamBreak 
+{
+    private DigitalInput initialConveyorSensor;
+    private int pinNum;
+    public NFRBeamBreak(int pinNum)
+    {
+        initialConveyorSensor = new DigitalInput(pinNum);
+        this.pinNum = pinNum;
+    }
+    public boolean beamIntact()
+    {
+        return initialConveyorSensor.get();
+    }
+    public boolean beamBroken()
+    {
+        return !initialConveyorSensor.get();
+    }
+
+    public BeamStatus GetStatus()
```

Comment:
Should probably be getStatus() instead of GetStatus() as the Java standard for naming implies all lowercase starting for method names.

Reply:
Right, I totally forgot about some of Java's interesting arbitrary formatting. Thank you.

Reply:
Okay, I will make the changes. Btw, this is Everett, but my username is pretty funky. Hope you're feeling better.

Reply:
Yeah, I was just about to comment to get the name right. I know zach's username, just for some reason I can also see him using Daboss-1 as a username. Thanks for your good wishes!

Reply:
Of course! I have to do some family things first, but then I will change the things wanted.